src/Standards/PSR2/Sniffs/Namespaces/UseDeclarationSniff.php

```diff
@@ -137,12 +137,19 @@ public function process(File $phpcsFile, $stackPtr)
             return;
         }
 
-        $end = $phpcsFile->findNext(T_SEMICOLON, ($stackPtr + 1));
+        $end         = $phpcsFile->findNext(T_SEMICOLON, ($stackPtr + 1));
+        $nextComment = $phpcsFile->findNext(T_COMMENT, ($end + 1));
```

Comment:
Should this be `Tokens::$emptyTokens` to allow for `phpcs:ignore`, etc.?

Reply:
Side note: What we are checking for here is the next inline comment, if that's on the same line as the `use` declaration then that can be classed as the "end" of the `use` declaration instead of the semicolon.

As I see it: Allowing for any "empty token" would mean a `/*` could be allowed (which it shouldn't be IMO) and other such tokens that shouldn't or couldn't be there.

I'm happy to be told that it **should** use `Tokens::$emptyTokens` I'm just of the opinion that the only thing that can come next should be an inline comment.

Reply:
`3.2.0` changed how overrides are annotated, which added several tokens for comments containing `phpcs:` overrides: https://github.com/squizlabs/PHP_CodeSniffer/releases/tag/3.2.0

My PR changed `T_COMMENT` to `Tokens::$emptyTokens` in order to allow the `T_PHPCS_*` tokens. That also allows `T_DOC_COMMENT_*`, which seemed safe but isn't 100% correct in hindsight. I think this should be an array with `T_COMMENT` and `T_PHPCS_*`.

@jrfnl Does that sound right to you?

Reply:
I'll have a look at this PR later today.